config/app.php

```diff
@@ -237,4 +237,10 @@
 
     ],
 
+    'auth_roles_enable' =>  (bool) env('AUTH_ROLES_ENABLE', false),
+    'auth_roles_header' =>  env('AUTH_ROLES_HEADER', 'remote-groups'),
+    'auth_roles_http_header' =>  env('AUTH_ROLES_HTTP_HEADER', 'HTTP_REMOTE_GROUPS'),
+    'auth_roles_admin' =>  env('AUTH_ROLES_ADMIN', 'admin'),
+    'auth_roles_delimiter' =>  ',',
```

Comment:
This also have to be an environment variable. I'm using Authentik and the delimiter is "|".

Reply:
Fixed with [6a60cdf](https://github.com/linuxserver/Heimdall/pull/1304/commits/6a60cdff966f24b197bfadb11d7624f5cf341b0c)